editions/tw5.com/tiddlers/filters/limit.tid

```diff
@@ -8,12 +8,15 @@ purpose: select the first or last <<.place n>> input titles
 
 <$macrocall $name=".operator-def"
 input="a [[selection of titles|Title Selection]]"
+suffix="optional, an offset"
```

Comment:
For consistency with the documentation of the other filter operators, this should say:

suffix="optionally, an offset, defaulting to 0"

I've added another note below about whether "offset" is the clearest term.